Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -4004,7 +4004,7 @@ void MainWindow::slotClientExit(bool /*checked =false */)
 
 void MainWindow::slotMeChangeNickname(bool /*checked =false */)
 {
-    QString nick = ttSettings->value(SETTINGS_GENERAL_NICKNAME, QCoreApplication::translate("MainWindow", SETTINGS_GENERAL_NICKNAME_DEFAULT)).toString();

```

Comment:
Keep SETTINGS_GENERAL_NICKNAME_DEFAULT but set SETTINGS_GENERAL_NICKNAME_DEFAULT to an empty string